report-jtreg-comparator/src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/Options.java

```diff
@@ -35,6 +39,10 @@ public Options() {
         this.hidePasses = false;
         this.die = false;
         this.configurations = new HashMap<>();
+        this.referentialJobName = null;
+        this.referentialBuildNumber = -1;
+        this.substringSide = Side.Tail;
+        substringLength = 5000; // TODO this default value is probably subject to change
```

Comment:
oh here it is.. jsut  not yet customized. ok Sorry. The "todo"  GIANTIC DIV from jenkins parent was making this invisible.

Reply:
Yes, here is the default.
But you can set custom size with switch `--cut-trace "tail:30000"` for example

Reply:
I should add it to help :)

Reply:
right :)

Reply:
> Yes, here is the default. But you can set custom size with switch `--cut-trace "tail:30000"` for example

great idea! TY!